verification/curator-service/ui/src/components/Users.test.tsx

```diff
@@ -37,7 +37,14 @@ test('lists users', async () => {
         config: {},
         headers: {},
     };
-    mockedAxios.get.mockResolvedValueOnce(axiosResponse);
+    mockedAxios.get.mockImplementation((url) => {
+        if (url === '/api/users/roles') {
+            return Promise.resolve({
+                data: { roles: ['admin', 'curator', 'reader'] },
```

Comment:
this promise data is similar in all mocked calls, consider refactoring it

Reply:
Done